object/virtual_device_list.go

```diff
@@ -57,6 +57,18 @@ func SCSIControllerTypes() VirtualDeviceList {
 	})
 }
 
+// SATAControllerTypes are used for adding a new SATA controller to a VM.
+func SATAControllerTypes() VirtualDeviceList {
+	// Return a mutable list of SATA controller types, initialized with defaults.
+	return VirtualDeviceList([]types.BaseVirtualDevice{
+		&types.VirtualSATAController{},
```

Comment:
If this function were used, should also have (or only have?) `VirtualAHCIController` in this list?

Reply:
Resolved in 2e77836.